tests/unit/amo/components/TestDropdownMenu.js

```diff
@@ -93,9 +93,13 @@ describe(__filename, () => {
   });
 
   it('sets active on mouseEnter/clears on mouseLeave', () => {
+    const _window = {
```

Comment:
If you want to add coverage that this was actually called, you could add an assertion about it in the test (the same goes for the next test), but I don't think that's necessary as we can see from the tests that it does in fact impact the behaviour of the component.

Reply:
50b07cc3d4839b71ecce6f9076387771a0b0d5bf just in case